MCP_Server/server.py

```
@@ -652,10 +667,493 @@ def load_drum_kit(ctx: Context, track_index: int, rack_uri: str, kit_path: str)
+        results = []
+        
+        # Apply preset settings
+        for band_index, settings in preset.items():
+            # Convert band_index (0-7) to the actual band number (1-8)
+            band_number = band_index + 1
+            
+            # Enable/disable the band
+            if "enabled" in settings:
+                enable_param_name = f"{band_number} Filter On A"
+                enable_value = 1 if settings["enabled"] else 0
+                enable_result = ableton.send_command("set_device_parameter", {
```

Ruff (0.8.2)

Local variable `enable_result` is assigned to but never used

Remove assignment to unused variable `enable_result`

(F841)